packages/flutter_tools/lib/src/ios/simulators.dart

```diff
@@ -26,12 +27,29 @@ class IOSSimulators extends PollingDeviceDiscovery {
   IOSSimulators() : super('IOSSimulators');
 
   bool get supportsPlatform => Platform.isMacOS;
-  List<Device> pollingGetDevices() => IOSSimulator.getAttachedDevices();
+  List<Device> pollingGetDevices() => IOSSimulatorUtils.instance.getAttachedDevices();
+}
+
+class IOSSimulatorUtils {
+  /// Returns [IOSSimulatorUtils] active in the current app context (i.e. zone).
+  static IOSSimulatorUtils get instance => context[IOSSimulatorUtils] ?? (context[IOSSimulatorUtils] = new IOSSimulatorUtils());
```

Comment:
That, and also if you are importing `Foo` explicitly via `show`, then `Foo` is the only thing you need to `show`:

```dart
import 'simulators.dart' show IOSSimulatorUtils;

main() {
  IOSSimulatorUtils utils = IOSSimulatorUtils.instance;
}
```

I'm also planning to do a PR to clean up `globals.dart`. Otherwise it will grow into reexporting everything in the world.